python/dask_cudf/dask_cudf/tests/test_binops.py

```diff
@@ -62,7 +64,7 @@ def test_series_binops_integer(binop):
     rhs_df, rhs_gdf = _make_random_frame(size)
     got = binop(lhs_gdf.x, rhs_gdf.y)
     exp = binop(lhs_df.x, rhs_df.y)
-    dd.assert_eq(got, exp)
+    dd.assert_eq(got, upcast(exp))
```

Comment:
Maybe we write our own `assert_eq` local to `dask_cudf` that encapsulates this, rather than repeating this pattern in all our tests?

Reply:
I think that's reasonable 👍 will try it

Reply:
Looks much better. Do you think calling it `assert_dd_eq` instead of aliasing it at import would help to disambiguate it better from cudf's `assert_eq`? Sorry I didn't suggest this earlier, didn't envision this